### ISSUE TITLE
Hyperparameters for training

### ISSUE THREAD

**Issue by sjzhuo** [THUKey/pytorch-mobilenet-v3#2]
Would you please provide the hyperparameters used for training of the models? Thanks

**Comment by kuan-wang** [comment]
number of epochs: 150
learning rate schedule: cosine learning rate, initial lr=0.05
weight decay: 4e-5
remove dropout

**Comment by AbdallaGomaa** [comment]
Could you please provide the batch size you used? Thanks!

**Comment by kuan-wang** [comment]
batch_size = 256
